app/controllers/provider/admin/account/change_plans_controller.rb

```diff
@@ -5,7 +5,7 @@ def show
   end
 
   def update
-    plan = plans.find(params[:change_plan][:plan_id])
+    plan = plans.find(params.permit(:change_plan)[:plan_id])
```

Comment:
You have to permit the subkeys too

See https://api.rubyonrails.org/v5.0.2/classes/ActionController/Parameters.html

Reply:
@hallelujah permitted the used ones